hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -26,10 +26,22 @@ def fail_and_exit(message)
     exit 1
   end
 
+  def foreman_server?
+    module_enabled?('foreman')
+  end
+
   def local_postgresql?
     param_value('foreman', 'db_manage') || param_value('katello', 'candlepin_manage_db')
   end
 
+  def local_redis?
+    (foreman_server? && !param_value('foreman', 'jobs_sidekiq_redis_url')) || module_enabled?('pulpcore')
```

Comment:
We don't enable pulpcore as a module in the installer.
```suggestion
    (foreman_server? && !param_value('foreman', 'jobs_sidekiq_redis_url')) || param_value('foreman_proxy::plugin::pulp', 'pulpcore_enabled')
```

Reply:
Ahh yea, I was on autopilot writing this. Thanks!